docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+# First Steps with DataFrames.jl
+
+The first and important part to work with DataFrames.jl, is to install the `DataFrames`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ] # `]` should be pressed 
+
+(@v1.6) pkg>  add DataFrames
+```
+
+If you want to make sure everything works as expected you could run the tests bundled
+with DataFrames.jl, but be warned that it will take more than 30 minutes.
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Additionally, it is recommended to check the version installed with `st` command(this tutorial is for 1.x releases).
+```julia
+julia> ]
+
+(@v1.6) pkg> st 
+```
+
+Throughout the rest of the tutorial we will assume that you have installed DataFrames.jl package and
+have already typed `using DataFrames`.
+Let's get started by loading the DataFrames.jl package:
+```jldoctest
+julia> using DataFrames
+```
+
+The most fundamental object provided by DataFrames.jl is a `DataFrame`, where
+typically that each row is interpreted as an observation and each column as a feature.
+
+# Constructors and basic utility functions
+
+## Constructors
+
+In this section you will see many ways to create a `DataFrame` using the `DataFrame(...)` constructor.
+First, let's create an empty `DataFrame`:
+```jldoctest dataframe
+julia> using DataFrames
+
```

Comment:
```suggestion
```

Reply:
you already have said that you assume this has been executed. Same below.

Reply:
Yes, but last time documenter was throwing an error that DataFrame is not defined.

Reply:
Do you mean I should define like this?
```jldoctest dataframe

Reply:
Yes